CHANGELOG.md

```diff
@@ -16,6 +16,7 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 - Fix error logged by Jaeger remote sampler on empty or unset `OTEL_TRACES_SAMPLER_ARG` environment variable (#6511)
 - Relax minimum Go version to 1.22.0 in various modules. (#6595)
+- `NewSDK` handles `OpenTelemetryConfiguration.Resource` properly in `go.opentelemetry.io/contrib/config/v0.3.0`. (#6606)
```

Comment:
This is a bit vague. Can we be explicit and maybe say this fixes the nil panic and handles empty resources?

Reply:
LGTM :+1:

Reply:
Either, look good to me :+1: